### PR TITLE
MINOR: Updating files with release 2.7.1

### PR DESCRIPTION
**Pull request by mimaison**
We need to wait for the artifact to show up in Maven before merging

Also I don't have permissions to push the artifacts to our S3 bucket. Can someone do it for me? Thanks

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: tests/docker/Dockerfile [hunk 1]
```diff
@@ -61,7 +61,7 @@ RUN mkdir -p "/opt/kafka-2.3.1" && chmod a+rw /opt/kafka-2.3.1 && curl -s "$KAFK
 RUN mkdir -p "/opt/kafka-2.4.1" && chmod a+rw /opt/kafka-2.4.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.4.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.4.1"
 RUN mkdir -p "/opt/kafka-2.5.1" && chmod a+rw /opt/kafka-2.5.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.5.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.5.1"
 RUN mkdir -p "/opt/kafka-2.6.2" && chmod a+rw /opt/kafka-2.6.2 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.6.2.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.6.2"
-RUN mkdir -p "/opt/kafka-2.7.0" && chmod a+rw /opt/kafka-2.7.0 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.7.0.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.7.0"
+RUN mkdir -p "/opt/kafka-2.7.1" && chmod a+rw /opt/kafka-2.7.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.7.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.7.1"
```
Comment (chia7712) [inline tests/docker/Dockerfile:64]: the link (https://s3-us-west-2.amazonaws.com/kafka-packages/kafka_2.12-2.7.1.tgz) is not available now. Could you fix it?

**Review by chia7712 — COMMENTED** [review]
@mimaison thanks for this patch. one small comment below.

File: tests/docker/Dockerfile (hunk 1 above)
Comment (mimaison) [reply tests/docker/Dockerfile:64]: See my comment in https://github.com/apache/kafka/pull/10660#issue-637378228

**Comment by mimaison** [comment]
@mjsax @vvcephei can you take a look? Thanks

**Comment by mjsax** [comment]
> Also I don't have permissions to push the artifacts to our S3 bucket.

Yeah. Confluent own this bucket... We need to find a better solution for this... uploaded both artifacts. Can you double check that you can read them?

**Comment by mjsax** [comment]
Build failed with `Could not find org.apache.kafka:kafka-streams:2.7.1.` -- we should try to run it again later and only merge after it passed to be sure the artifact are accessbile.

**Comment by mimaison** [comment]
Thanks @mjsax, I can see the artifacts now. I've rekicked the build

**Comment by mimaison** [comment]
@mjsax / @chia7712  Can you take another look? Thanks

**Comment by mimaison** [comment]
@rajinisivaram @omkreddy Can you take a look? We need this PR and https://github.com/apache/kafka-site/pull/354 in order to release 2.7.1. Thanks

**Review by omkreddy — APPROVED** [review]
@mimaison Thanks for the PR. LGTM

**Pull request merged**